src/steps/mdx-to-gridtables.ts

```diff
@@ -125,7 +111,9 @@ export default function mdxToBlocks(ctx: Helix.UniversalContext) {
       ],
     }));
 
-    const totalSlots = repeat * slots.length;
+    // Determine how many children to treat as slots
+    const totalSlots = slotsValue === 'none' ? 0 : repeat * slots.length;
```

Comment:
can we remove the === 'none' if we are not using that? and just use the slots.length to count?

Reply:
just continue using the totalSlots as what is it previously.